plugins/flatpak/gs-flatpak.c

```diff
@@ -3418,8 +3423,41 @@ gs_flatpak_app_get_copyable (GsFlatpak *self,
 			     GCancellable *cancellable,
 			     GError **error)
 {
-	if (copyable)
-		*copyable = (gs_flatpak_get_collection_id (self, app) != NULL);
+	if (copyable) {
+		GsApp *runtime;
+		g_autoptr(FlatpakRemote) app_remote = NULL;
+		g_autoptr(FlatpakRemote) runtime_remote = NULL;
+		g_autofree char *app_collection_id = NULL;
+		g_autofree char *runtime_collection_id = NULL;
+		gboolean is_copyable = TRUE;
+
+		/* Check if the app's remote has a collection ID set */
+		app_remote = flatpak_installation_get_remote_by_name (self->installation,
```

Comment:
As discussed in Slack, I'd prefer to invert the logic such that `is_copyable = (app_remote != NULL && app_collection_id != NULL && ...)`, to play it safe. That is, a Flatpak needs to be in a reasonable state/have reasonable metadata to support it being copied. This reduces exposure to issues like the crasher this patch solves and we can test and enable more-permissive branches if and when we find they're necessary and safe.